src/main/java/com/github/timm/cucumber/generate/CucumberITGeneratorByFeature.java

```diff
@@ -67,9 +74,9 @@ private void initTemplate() {
             props.put("file.resource.loader.path", config.getProjectBasedir().getAbsolutePath());
             name = config.getCustomVmTemplate();
         } else if (config.useTestNG()) {
-            name = "cucumber-testng-runner.vm";
+            name = "cucumber-testng-runner.java.vm";
         } else {
-            name = "cucumber-junit-runner.vm";
+            name = "cucumber-junit-runner.java.vm";
```

Comment:
Changing the extension to .java.vm ensures idea can do proper syntax highlighting.